distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/strategy/EvenlySpreadAssignmentStrategy.java

```diff
@@ -0,0 +1,62 @@
+package ru.fix.distributed.job.manager.strategy;
+
+import ru.fix.distributed.job.manager.model.AssignmentState;
+import ru.fix.distributed.job.manager.model.JobId;
+import ru.fix.distributed.job.manager.model.WorkItem;
+import ru.fix.distributed.job.manager.model.WorkerId;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class EvenlySpreadAssignmentStrategy extends AbstractAssignmentStrategy {
+
+    @Override
+    public AssignmentState reassignAndBalance(
+            Map<JobId, Set<WorkerId>> availability,
+            AssignmentState prevAssignment,
+            AssignmentState currentAssignment,
+            Set<WorkItem> itemsToAssign
+    ) {
+        for (Map.Entry<JobId, Set<WorkerId>> jobEntry : availability.entrySet()) {
+            Set<WorkItem> itemsToAssignForJob = getWorkItemsByJob(jobEntry.getKey(), itemsToAssign);
```

Comment:
jobEntry.getKey and jobEntry.getValue all over the place are very hard to read.
It would be better to give clear names:
```
for(Map<> jobAvailability: availability.entrySet()) {
    JobId job = jobEntry.getKey();
    Set<WorkerId> availableWorkers = jobEntry.getValue();
    
     ... logic that uses job and availableWorkers 
    
}
```

Reply:
done